File: TennisLeague.API/Models/Facility.cs
namespace TennisLeague.API.Models
{
    public class Facility
    {
        public int? ID { get; set; } = null;
        public string Name { get; set; } = string.Empty;
        public string AddressLine1 { get; set; } = string.Empty;
        public string? AddressLine2 { get; set; } = null;
        public string City { get; set; } = string.Empty;
        public string State { get; set; } = string.Empty;
        public string Zip { get; set; } = string.Empty;
        public string Phone { get; set; } = string.Empty;
        public string Email { get; set; } = string.Empty;
        public int NumberOfCourts { get; set; } = 0;
    }
}

File: TennisLeague.Web/ClientApp/src/models/Facility.ts
export class Facility {
    constructor() {
        this.name = '';
        this.addressLine1 = '';
        this.city = '';
        this.state = '';
        this.zip = '';
        this.numberOfCourts = 0;
    }

    id?: number;
    name: string;
    addressLine1: string;
    city: string;
    state: string;
    zip: string;
    numberOfCourts: number;
}